### PR TITLE
Add gravatar and make pretty

### PR DESCRIPTION
**Pull request by ericworkman**
@mbramson as it says. Here are some screenshots:

Desktop:
screen shot 2017-02-04 at 5 49 36 pm

IPhone 6 Plus
screen shot 2017-02-04 at 5 49 42 pm

### CODE REVIEW

**Comment by ericworkman** [comment]
I've made the /pair route prettier too. I expect to add more to the user panel over time.

screen shot 2017-02-04 at 7 27 31 pm


screen shot 2017-02-04 at 7 27 49 pm
screen shot 2017-02-04 at 7 27 56 pm

File: web/templates/profile/show.html.eex [hunk 1]
```diff
@@ -1,46 +1,48 @@
-<h2>User Profile</h2>
```
Comment (mbramson) [inline web/templates/profile/show.html.eex:1]: Why the decision to drop this off?

**Review by mbramson — COMMENTED** [review]
I feel like the size of the new icons on the /pairs route sort of deemphasizes the aspects of the page that make the fact that you're paired pop. Does that makes sense? I feel like its harder at first glance to figure out that there's actually pairing going on.

Perhaps smaller icons that are to the left of the user's name would keep the icons from overpowering the rest of the design?

This is awesome! Great idea!

File: web/templates/profile/show.html.eex (hunk 1 above)
Comment (ericworkman) [reply web/templates/profile/show.html.eex:1]: I felt it was redundant. I could keep it as `Profile` if you'd like.

**Comment by ericworkman** [comment]
@mbramson like this better?

screen shot 2017-02-05 at 11 34 38 am
screen shot 2017-02-05 at 11 34 33 am

**Comment by mbramson** [comment]
Yep, that looks solid. LGTM

**Review by mbramson — APPROVED** [review]
LGTM

**Pull request merged**